### DISCUSSION TITLE
[SOLVED] Improvements in vertical layout of browser

### DISCUSSION THREAD

**Discussion by Starboy-Xo** [martinrotter/rssguard#600] [General]
## Please take a look into the following picture:

![image](https://user-images.githubusercontent.com/15247707/150319583-2aeb1558-5d9c-4890-b7db-6c8185d4085e.png)

## This CONCEPT photoshopped pic may help:
![image](https://user-images.githubusercontent.com/15247707/150319999-b834f5c2-bbd5-49ed-a94d-19b3e7cecdae.png)

## The RED arrow Down button may replace URL link at the end of the message title. It may say, Open article here.
Best regards

**Comment by martinrotter** [comment]
For these obvious FR requests, create ISSUES, not discussions. Use discussions only for some hypothethic/broad discussion requests, ideas etc.

Converting to issue.

**Reply by Starboy-Xo** [reply]
Ok, I will keep it in mind next time.
